CHANGELOG.md

```diff
@@ -24,18 +24,19 @@
 - Remove coveralls.
 - Disable cache in travis.
 - Other refactoring.
+- more details about the transition can be found in #62
```

Comment:
I think most of the info from #62 should be migrated here.

If we still need any issue references I think any issue references, I think they should be links. Just using issue number reference does not seem to work like issues & PRs.